docs/_includes/lightbox-dialog.html

```diff
@@ -104,6 +104,66 @@ <h2 id="dialog-title" class="large-text-1 bold-text">Heading</h2>
 </div>
     {% endhighlight %}
 
+    <h3 id="lightbox-wide">Previous button in header Lightbox</h3>
+
+
```

Comment:
We should include a blurb about this variant (like is there for others) to describe it a bit and its usage.